Jenkinsfile

```diff
@@ -43,8 +40,8 @@ pipeline {
                     echo "[ Python used ] : " `which python`
                     cd ${env.WORKSPACE}
                     echo "[ Installing ${env.JOB_NAME} ]"
-# full installation is not possible as it takes more than 3.0Gb for building
-#                    python setup.py install
+# full installation is not possible as it takes more than 3.0Gb for building and kills the server
+#                   python setup.py install
                     python setup.py build_ext --inplace
 		            python konno_setup.py
```

Comment:
That's odd that you can create the konno matrices manually but not from the setup.py script?

Reply:
konno_setup.py has a limited depth (num_to_gen=5), just enough for auto testing. 
Konno matrices built by setup.y runs in at memory alloc failure (2.0Gib) at 16385